server/src/test/scala/controllers/application/DropInsControllerSpec.scala

```diff
@@ -1,17 +1,31 @@
 package controllers.application
 
+import akka.actor.ActorSystem
 import email.GovNotifyEmail
 import module.DRTModule
 import org.scalatestplus.mockito.MockitoSugar
 import org.scalatestplus.play.PlaySpec
 import play.api.test.Helpers._
 import play.api.test._
 import uk.gov.homeoffice.drt.crunchsystem.DrtSystemInterface
+import uk.gov.homeoffice.drt.testsystem.TestDrtSystem
+import akka.stream.Materializer
+import org.scalatest.BeforeAndAfterAll
+
+import scala.concurrent.ExecutionContext
+class DropInsControllerSpec extends PlaySpec with MockitoSugar with BeforeAndAfterAll {
```

Comment:
Add a line break between the imports and the class def. Intellij should be able to do this if you ask it to format the file